packages/@okta/vuepress-site/guides/protect-your-api/before-you-begin/springboot/create-app.md

```diff
@@ -1,3 +1,17 @@
-The [Spring Boot Getting Started Guide](https://spring.io/guides/gs/spring-boot/) teaches you the basics of building a Spring Boot application.
+The [Spring Boot Getting Started Guide](https://spring.io/guides/gs/spring-boot/) teaches you the basics of building a Spring Boot application. You can create a new Spring Boot project by visting the [Spring Initializr](https://start.spring.io) and selecting the dependencies **Web**, **Security**, and **Okta**, and then clicking **Generate Project** to download a zip file.
```

Comment:
Is [Spring **Initializr**] correct?

Reply:
yup :) don't ask me how they came up with the name though